editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Portal.tsx

```diff
@@ -12,84 +12,82 @@ import { getParentId } from "../lib/utils";
 import Hanger from "./Hanger";
 import Question from "./Question";
 
-const ExternalPortal: React.FC<any> = React.memo(
-  (props) => {
-    const copyNode = useStore((state) => state.copyNode);
-
-    const { data } = useQuery(
-      gql`
-        query GetFlow($id: uuid!) {
-          flows_by_pk(id: $id) {
+const ExternalPortal: React.FC<any> = (props) => {
```

Comment:
`React.memo()` was caching this component and causing it to update in an unexpected manner.

There's nothing particularly "expensive" going on here apart from the query. By adding an `id` field to the query response though Apollo will just cache this for us in the `InMemoryCache` and stop the query being repeated anyway. 

The big diff here is a bit weird it's the just the indentation of `React.memo()` being removed.